g11n-ws/vip-manager-l10n-agent/src/main/java/com/vmware/l10agent/conf/PropertyConfigs.java

```diff
@@ -197,4 +200,8 @@ public String getSyncListPath() {
 	public int getSyncBatchSize() { return (syncBatchSize - 1); }
 
 	public boolean isSyncBatchEnable() { return syncBatchEnable; }
+
+	public int getSyncReqBodySize(){
+		return (1024*1024*Integer.valueOf(this.reqBodySizeStr.toUpperCase().replaceAll("M", "").trim())) - (512*1024);
```

Comment:
it's better put the result of calculation to a variable so that it won't perform the calculation for each function call.

Reply:
OK